site/styles/_carousel.scss

```diff
@@ -97,15 +97,24 @@ s72-carousel {
   @include media-breakpoint-up(lg) {
     flex-direction: row;
     padding: var(--carousel-bottom-gradient-height) 50px 1px;
-  }
 
-  .sponsor {
-    margin-bottom: 8px;
-    @include media-breakpoint-up(lg) {
+    .sponsor-awards-wrapper {
       align-self: flex-end;
       margin-left: auto;
       order: 1;
     }
+
+    .sponsor {
+      text-align: right;
+    }
+
+    .meta-awards {
+      justify-content: end;
+    }
+  }
+
+  .sponsor {
+    margin-bottom: 10px;
```

Comment:
This should be a 24px margin-bottom and the award element should have the padding removed.

Reply:
- changed to 24px margin-bottom for carousel.
- padding no longer applied on carousel award